source/jsonnet/northern-ireland/lib/common_rules.libsonnet

```diff
@@ -69,6 +69,16 @@
       },
     },
   },
+  nisraOver5: {
+    id: 'date-of-birth-answer',
+    condition: 'less than or equal to',
+    date_comparison: {
+      value: '2020-07-01',
```

Comment:
The logic works, but the date on the card is 1 July 2021, so one is wrong

Reply:
I used existing nisraUnder4 which might have a wrong date attached, I need to double check.

Reply:
Ok, so it was supposed to be 2021, great catch, many thanks!